recipe/meta.yaml

```diff
@@ -13,14 +13,26 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
The relevant bit appears to be that conda-build isn't finding and updating the prefix in pip-constructed entrypoints:

```
File "c:\miniconda\conda-bld\recipe_1484670624752\_b_env\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\miniconda\conda-bld\recipe_1484670624752\_b_env\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Miniconda\conda-bld\recipe_1484670624752\_t_env\Scripts\nbdime.exe\__main__.py", line 5, in <module>
  File "c:\miniconda\conda-bld\recipe_1484670624752\_b_env\lib\site-packages\nbdime\__init__.py", line 10, in <module>
```

(there's a mixture of build and test env paths in there)